package.json

```diff
@@ -162,9 +162,9 @@
 		"moment": "2.29.4",
 		"objectFitPolyfill": "2.3.5",
 		"polyfill-library": "4.8.0",
-		"react": "18.2.0",
-		"react-dom": "18.2.0",
-		"react-is": "18.2.0",
+		"react": "18.3.1",
+		"react-dom": "18.3.1",
+		"react-is": "18.3.1",
```

Comment:
Seems there's a unit test failure because of this update. It think script-loader.php need to use the new version for the cache key. 

Also cc @Mamaduka as he was looking at doing this separately.

Reply:
I have already updated it, let's see if it passes now

Reply:
Hm, not sure why my commits are not appearing here, I pushed the change

Reply:
Why did this bot delete my branch?

<img width="923" alt="Screenshot 2024-05-23 at 17 06 14" src="https://github.com/WordPress/wordpress-develop/assets/4710635/6044f851-eb5c-43e8-a2f9-774a841735ff">

Reply:
Not sure about the commits but I noticed that you created a branch on Wordpress-develop directly, you should create a branch on your fork instead (and a new PR) because otherwise, this PR will be auto-closed.

Reply:
Oh

Reply:
using branches in this repository is not really allowed.

Reply:
:disappears:

Reply:
Thanks, @ellatrix! React update looks correct.